actionpack/lib/action_controller.rb

```diff
@@ -15,7 +16,6 @@ module ActionController
   autoload :FormBuilder
 
   autoload_under "metal" do
-    autoload :Compatibility
```

Comment:
The removal of that line is unrelated, but anyway it's a module that does not exist anymore.